### PR TITLE
Add SWIFT, Flamingo, MUSIC, and pNbody

### PR DESCRIPTION
**Pull request by mladenivkovic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -806,6 +806,13 @@ nbodykit is an open source project written in Python that provides a set of stat
 
 [nbodykit Website](https://nbodykit.readthedocs.io/en/latest/index.html)
 
+
+### pNbody
+
+pNbody is a parallelized python module toolbox designed to manipulate and display interactively very large N-body systems. Its oriented object approach allows the user to perform complicate manipulation with only very few commands. Some additional packages like [glups](https://gitlab.com/revaz/glups) (Python/OpenGL GUI designed to display Nbody models), [Gtools](https://gitlab.com/revaz/Gtools) (a suite of tools and scripts designed to reduce n-body systems representing galaxies) and [Mtools](https://gitlab.com/revaz/Mtools) (routines to adapt images for movies).
```
Comment (lonbar) [inline README.md:812]:
```suggestion
pNbody is a parallelized python module toolbox designed to manipulate and display interactively very large N-body systems. Its oriented object approach allows the user to perform complicated manipulations with only very few commands. Some additional packages like [glups](https://gitlab.com/revaz/glups) (Python/OpenGL GUI designed to display N-body models), [Gtools](https://gitlab.com/revaz/Gtools) (a suite of tools and scripts designed to reduce N-body systems representing galaxies) and [Mtools](https://gitlab.com/revaz/Mtools) (routines to adapt images for movies).
```

File: README.md [hunk 2]
```diff
@@ -1373,6 +1391,13 @@ Cosmological Adaptive Mesh Refinement code featuring N-body dynamics, magneto-hy
 
 https://bitbucket.org/rteyssie/ramses
 
+### SWIFT
+Open-source cosmological and astrophysical numerical solver designed to run efficiently on modern hardware. A comprehensive and extensive set of models for galaxy formation as well as planetary physics are provided alongside a large series of examples. Features N-body dynamics, particle-based hydrodynamics, a set of galaxy formation models and sub-grid models, radiative transfer, neutrinos, planetary science (equations of state) , line-of-sight outputs, light cone outputs, halo finding, on-the-fly power spectra computation.
```
Comment (lonbar) [inline README.md:1395]:
```suggestion
A open-source cosmological and astrophysical numerical solver designed to run efficiently on modern hardware. A comprehensive and extensive set of models for galaxy formation as well as planetary physics are provided alongside a large series of examples. Features N-body dynamics, particle-based hydrodynamics, a set of galaxy formation models and sub-grid models, radiative transfer, neutrinos, planetary science (equations of state), line-of-sight outputs, light cone outputs, halo finding, on-the-fly power spectra computation.
```

File: README.md [hunk 3]
```diff
@@ -1100,6 +1112,12 @@ The Magneticum simulations are aiming to follow the formation of cosmological st
 
 [Magneticum Publications](http://www.magneticum.org/publications.html)
 
+### Flamingo simulations
+
+Full-hydro Large-scale structure simulations with All-sky Mapping for the Interpretation of Next Generation Observations. Large cosmological simulations including baryonic matter, sub-grid physics, and neutrinos.
```
Comment (lonbar) [inline README.md:1117]:
```suggestion
The Full-hydro Large-scale structure simulations with All-Sky Mapping for the Interpretation of Next Generation Observations. Large cosmological simulations including baryonic matter, sub-grid physics, and neutrinos.
```

Comment (lonbar) [reply README.md:1117]: Do I understand correctly that FLAMENGO is a project that uses software, but is not itself software? @nikosarcevic is this in the scope of this project?

File: README.md [hunk 4]
```diff
@@ -856,6 +863,11 @@ The project is currently still under active development, so check the documentat
 
 [Repository](https://bitbucket.org/ohahn/monofonic/src/master/)
 
+### MUSIC
+MUSIC generates nested grid initial conditions for high-resolution "zoom" cosmological simulations with outputs for RAMSES, ENZO, Arepo, Gadget2/3, Pkdgrav/Gasoline, and Nyx. Supports first (1LPT) and second order (2LPT) Lagrangian perturbation theory, local Lagrangian approximation (LLA) for baryons with grid codes. Supports pluggable transfer functions, currently CAMB, Eisenstein&Hu, BBKS, Warm Dark Matter variants. Distinct baryon+CDM fields. Parallelized with OpenMP.
```
Comment (lonbar) [inline README.md:867]:
```suggestion
The Multi-Scale Initial Conditions for Cosmological Simulations (MUSIC) generates nested grid initial conditions for high-resolution "zoom" cosmological simulations with outputs for RAMSES, ENZO, Arepo, Gadget2/3, Pkdgrav/Gasoline, and Nyx. Supports first (1LPT) and second order (2LPT) Lagrangian perturbation theory, local Lagrangian approximation (LLA) for baryons with grid codes. Supports pluggable transfer functions, currently CAMB, Eisenstein&Hu, BBKS, Warm Dark Matter variants. Distinct baryon+CDM fields. Parallelized with OpenMP.
```

Comment (lonbar) [reply README.md:867]: Can you add links/references to the various projects and transfer functions listed here?

File: README.md (hunk 1 above)
Comment (lonbar) [reply README.md:812]: > Some additional packages like [..]

I don't quite understand what this means (it is not a complete sentence). Are some additional packages available? Are they plugins for pNbody? Are there other packages/plugins?

**Review by lonbar — COMMENTED** [review]
Thanks for this contribution! I have made some minor suggestions for improvement. I am not sure if FLAMENGO is in the scope of this project (@nikosarcevic can you verify?)

File: README.md (hunk 3 above)
Comment (mladenivkovic) [reply README.md:1117]: It's a simulation suite, comparable to Illustris, IllustrisTNG, Magneticum, Auriga  (all of which are listed there, which is why I added Flamingo)

File: README.md (hunk 4 above)
Comment (mladenivkovic) [reply README.md]: Not sure what you'd like me to link/reference here. It's all part of the repo?

File: README.md (hunk 1 above)
Comment (mladenivkovic) [reply README.md]: Sorry, my bad. It's been a long day. Better?

Comment (lonbar) [reply README.md]: Perfect, thanks for clarifying.

**Review by lonbar — APPROVED** [review]

**Pull request merged**